ledger/complete/ledger.go

```diff
@@ -265,7 +267,8 @@ func (l *Ledger) Checkpointer() (*wal.Checkpointer, error) {
 func (l *Ledger) ExportCheckpointAt(
 	state ledger.State,
 	migrations []ledger.Migration,
-	reporters []ledger.Reporter,
+	reporters map[string]ledger.Reporter,
+	extractionReportName string,
```

Comment:
same here, it seems this to be related to a single reporter and not something ledger package should be aware of